### ISSUE TITLE
Deprecated Twig class will be removed in Twig 2.0

### ISSUE THREAD

**Issue by hmillet** [suncat2000/MobileDetectBundle#75]
Hi,

my symfony 2 app start to warn me about usage of deprecated method in this bundle :

```
DEPRECATED - The Twig_Function_Method class is deprecated since version 1.12 and will be removed in 2.0. Use Twig_SimpleFunction instead.

DEPRECATED - Using an instance of "Twig_Function_Method" for function "is_mobile" is deprecated. Use Twig_SimpleFunction instead.
```

Will someone take care of this and make a new release for Twig 2.0 ?

Thx

**Comment by xabbuh** [comment]
see #77 (the `Twig_Filter_Method` class must be used in another of your dependencies)

**Comment by hmillet** [comment]
Thx @xabbuh 

In fact, I'm talking about Twig_Function_Method, that is used at line 55 and following in ```Twig/Extension/MobileDetectExtension.php```

**Comment by suncat2000** [comment]
fixed in #77
